ibl_to_nwb/brainwide_map/read_after_write_testing.py

```diff
@@ -0,0 +1,44 @@
+import os
+import numpy as np
+import pandas as pd
+from pathlib import Path
+import h5py
+from pynwb import NWBHDF5IO, NWBFile
+from one.api import ONE
+import logging
+import read_after_write as raw
```

Comment:
Highly recommend not aliasing to real words that also have other contextual meaning.

Also don't recommend importing as an improper local module

```suggestion

from ibl_to_nwb.brainwide_map import read_after_write
```

Reply:
good point